pl_examples/hydra_examples/conf/config.yaml

```diff
@@ -1,11 +1,14 @@
 defaults:
-  - model: basic
+  - model/data: mnist
+  - model/arch: basic
+  - model/lr_scheduler: null
+  - model/optimizer: sgd
   - pl/trainer: trainer
   - pl/logger: null
-  - pl/profiler: simple
+  - pl/profiler: null
   - pl/checkpoint: null
   - pl/early_stopping: null
 
 hydra:
```

Comment:
Not sure this one belongs here. we should keep this minimal.
If this is for the tensorboard logger, you might mention in a readme section about it.

Reply:
the profiler is part of pytorch lightning general configuration, I think we will see a simplification here once I develop the top level config for PL.

Reply:
ah, no. I was talking about configuring the Hydra output directory.

Reply:
Fixed